chcemvediet/apps/anonymization/anonymization.py

```diff
@@ -53,47 +53,51 @@
     u'ž': u'(?:z|ž)',
 }
 
-def generate_word_pattern(words):
+def generate_word_pattern(words, substring):
```

Comment:
Skor by som to nazval `match_subwords`, kedze slovo string je moc vseobecne. Prefix `match` by som dal koli tomu, aby to neevokovalo, ze parameter je nejaky substring, ale ze je to prepinac.

Imho toto:
```
{% anonymize inforequest foo match_subwords=True %}
```
je zrozumitelnejsie, co to znemana, nez toto
```
{% anonymize inforequest foo substring=True %}
```